### DISCUSSION TITLE
Multiple PCIs with same gNB

### DISCUSSION THREAD

**Discussion by mrahman2-vt** [srsran/srsRAN_Project#726] [General Help]
How to run multiple cells with same gNB like 3 sectors of a gNB?

**Comment by improperaffo** [comment]
Hi @mrahman2-vt, may I ask you why do you need three PCIs?

The configurations provided in [here](https://github.com/srsran/srsRAN_Project/tree/main/configs) are sufficient to connect multiple phones. You just need to select the one that matches your SDR.

Hope this can help,
Raphael

**Reply by mrahman2-vt** [reply]
I would like to connect multiple phones simultaneously, that's why need to have 3 different PCIs for 3 phones. Seems like I don't have access to that link, is it restricted? or does it still works?

**Reply by improperaffo** [reply]
My bad, there was a problem with the link. [Here](https://github.com/srsran/srsRAN_Project/tree/main/configs) is the correct one.

**Reply by andrepuschmann** [reply]
Hey @mrahman2-vt  - if you have a single cell you can connect more than just one UE/phone to that cell without any issues. Multiple PCIs are used if you want to server multiple cells/sectors.

**Reply by improperaffo** [reply]
@mrahman2-vt you can see some examples of gNB yaml file with multiple PCIs in the [gNB handover section](https://docs.srsran.com/projects/project/en/latest/tutorials/source/handover/source/index.html#configuration), but you don't need it for connecting multiple UEs.

What do you mean by "_none of them work_"? They cannot even connect? Did you look at the AMF logs?
